CI - skip sorting of imports with isort

pavoljuhas opened the pull request:
The `check/format-incremental` needs more work.
isort invocation should skip `__init__.py` files, but it does not.

Related to #7181

dstrain115 commented on check/format-incremental:
```diff
@@ -109,8 +109,9 @@ if (( only_print == 1 )); then
-isort "${args[@]}" "${format_files[@]}"
-ISORTSTATUS=$?
+echo "skipped"; ISORTSTATUS=0
+# isort "${args[@]}" "${format_files[@]}"
```
I would prefer not having commented code in script files.

pavoljuhas replied:
done in ac22abb00fe8edf9dd91c3d176583c49c8373a5a

pavoljuhas commented:
NB: this is a quick fix to prevent isort misbehaving in CI checks.

To be reverted after fixups in `check/format-incremental`

daxfohl commented:
What happened? It worked for me when I added the skip directive to the config.

pavoljuhas commented:
> What happened? It worked for me when I added the skip directive to the config.

The `skipped` directive is overruled if such file is present in isort arguments.
This would happen for `check/format-incremental --all` or if some `__init__.py` is
changed in a PR.

The `__init__.py` files should be excluded from isort, but should stay present for `black`.

daxfohl commented:
Oh I see. The skip directive only works when calling `isort .` but the script specifies which files to isort explicitly, which seems to override the config.
